### DISCUSSION TITLE
Questions about the overhead of blockchain in Swarm Learning

### DISCUSSION THREAD

**Discussion by daenerys7** [HewlettPackard/swarm-learning#109] [General]
Hi, I noticed that the platform running the Swarm Network nodes is GETH. I would like to know, how to estimate the overhead of the blockchain part such as ETH Gas during the Swarm Learning process. Or are there other metrics to measure this overhead?

**Comment by IMOKURI** [comment]
Currently, I do not believe there is a way to measure Blockchain overhead.

However, I think the latency to execute a transaction is very small. This is due to the low difficulty.

On the other hand, merging models requires time for model transfer.
This overhead depends on the network specs.

**Comment by iArpanPatel** [comment]
Converting this as discussion topic.
